transcript/src/transcript.rs

```diff
@@ -1,153 +1,166 @@
 use crate::{
-    halo2::{arithmetic::CurveAffine, plonk::Error},
+    halo2::{
+        arithmetic::{CurveAffine, FieldExt},
+        plonk::Error,
+    },
     hasher::HasherChip,
     maingate::{AssignedValue, RegionCtx},
 };
-use ecc::{halo2::circuit::Chip, AssignedPoint, BaseFieldEccChip};
+use ecc::{
+    halo2::circuit::Chip,
+    maingate::{big_to_fe, decompose, fe_to_big},
+    AssignedPoint, BaseFieldEccChip,
+};
+use group::ff::PrimeField;
 use poseidon::Spec;
-use std::marker::PhantomData;
 
 /// `PointRepresentation` will encode point with an implemented strategy
+/// TODO: Generalize `ecc_chip` with `EccInstrucitons`
```

Comment:
Should we file an issue for this and remove the `TODO`?

Reply:
Already filed the issue. Feel free to just remove this.

Reply:
Removed in https://github.com/privacy-scaling-explorations/halo2wrong/pull/38/commits/0ca396e35d51807a87de72e776a20081540808d4